Client/Frontend/Home/Wallpapers/Utilities/WallpaperStorageUtility.swift

```diff
@@ -130,4 +130,28 @@ class WallpaperStorageUtility: WallpaperFilePathProtocol, Loggable {
             browserLog.debug("WallpaperStorageUtility - error deleting folder for \(key)")
         }
     }
+
+    // MARK: - Migration
+    func migrateResources(completion: (Bool) -> Void) {
+        let fileManager = FileManager.default
+        guard let documentPath = fileManager.urls(for: .documentDirectory, in: FileManager.SearchPathDomainMask.userDomainMask).first,
+              let appSupportPath = fileManager.urls(for: .applicationSupportDirectory, in: FileManager.SearchPathDomainMask.userDomainMask).first
+        else {
+            browserLog.debug("Wallpaper migration error")
+            return
+        }
+
+        let wallpaperDocumentDirectoryPath = documentPath.appendingPathComponent("wallpapers")
+        let wallpaperAppSupportDirectoryPath = appSupportPath.appendingPathComponent("wallpapers")
+
+        do {
+            try fileManager.moveItem(at: wallpaperDocumentDirectoryPath,
```

Comment:
FYI - I have seen cases where sometimes a move can fail if the src and dst are not on the same container. This doesn't usually show up in the simulator, but on a real device App support directory and Documents Directory are conceptually separated. If this has already been tested on a real device then it should be OK, but you may want to add something to the catch block that tries doing a copy and remove as a fallback.